src/applications/disability-benefits/all-claims/pages/index.js

```diff
@@ -308,6 +308,11 @@ import {
   schema as hospitalizationHistorySchema,
 } from './hospitalizationHistory';
 
+import {
```

Comment:
Do `import * as medicalCare from './medicalCare';`

Reply:
That's really cool. Thank you!